tasks/request_files/request_files.py

```diff
@@ -235,88 +244,114 @@ def inner_task(event: Dict, max_retries: int, retry_sleep_secs: float,
-            db_queue_url: todo
-            job_id: The unique identifier used for tracking requests.
-
-        Raises: RestoreRequestError if any file restore could not be initiated.
+    Args:
+        s3: An instance of boto3 s3 client
+        granule: A dict with the following keys:
+            'granuleId' (str): The id of the granule being restored.
+            'recover_files' (list(dict)): A list of dicts with the following keys:
+                'key' (str): Name of the file within the granule.
+                'dest_bucket' (str): The bucket the restored file will be moved
+                    to after the restore completes
+                'success' (bool): Should enter this method set to False. Modified to 'True' by method end.
+                'err_msg' (str): Will be modified if error occurs.
+
+
+        glacier_bucket: The S3 glacier bucket name.
+        restore_expire_days:
+            The number of days the restored file will be accessible in the S3 bucket before it expires.
+        max_retries: todo
+        retry_sleep_secs: todo
+        retrieval_type: todo
+        db_queue_url: todo
+        job_id: The unique identifier used for tracking requests.
+
+    Raises: RestoreRequestError if any file restore could not be initiated.
     """
-    request_time = datetime.now(timezone.utc).isoformat()
     attempt = 1
     granule_id = granule[GRANULE_GRANULE_ID_KEY]
 
-    post_status_for_job_to_queue(job_id, granule_id, ORCA_STATUS_PENDING, request_time, None, glacier_bucket,
-                                 RequestMethod.POST,
-                                 db_queue_url,
-                                 max_retries, retry_sleep_secs)
-
     while attempt <= max_retries + 1:
         for a_file in granule[GRANULE_RECOVER_FILES_KEY]:
             if not a_file[FILE_SUCCESS_KEY]:
                 try:
-                    restore_object(s3, a_file[FILE_KEY_KEY], restore_expire_days, glacier_bucket, attempt, job_id,
-                                   retrieval_type)
-                    a_file[FILE_SUCCESS_KEY] = True
-                    a_file[FILE_ERROR_MESSAGE_KEY] = ''
-
-                    post_status_for_file_to_queue(
-                        job_id, granule_id, os.path.basename(a_file[FILE_KEY_KEY]),
+                    restore_object(
+                        s3,
                         a_file[FILE_KEY_KEY],
-                        a_file[FILE_DEST_BUCKET_KEY],
-                        ORCA_STATUS_PENDING,
-                        None,
-                        request_time,
-                        datetime.now(timezone.utc).isoformat(),
+                        restore_expire_days,
+                        glacier_bucket,
+                        attempt,
+                        job_id,
+                        retrieval_type,
+                    )
+                    a_file[FILE_SUCCESS_KEY] = True
+                    a_file[FILE_ERROR_MESSAGE_KEY] = ""
+
+                    files = [
```

Comment:
Is this the correct format for files? Looking at the schema for post to database for the input here https://github.com/nasa/cumulus-orca/blob/develop/tasks/post_to_database/schemas/new_job_input.json it looks like this should be this:

```
files = [
    {
        "filename": os.path.basename(a_file[FILE_KEY_KEY]),
        "key_path": a_file[FILE_KEY_KEY],
        "restore_destination": a_file[FILE_DEST_BUCKET_KEY],
        "status_id": shared_recovery.OrcaStatus.PENDING.value,
        "error_message": None,
        "request_time": datetime.now(timezone.utc).isoformat(),
        "last_update": datetime.now(timezone.utc).isoformat(),
       "completion_time": None
    },
]
```

Reply:
what I currently have: 

                    files = [
                        {"filename": os.path.basename(a_file[FILE_KEY_KEY])},
                        {"key_path": a_file[FILE_KEY_KEY]},
                        {"restore_destination": a_file[FILE_DEST_BUCKET_KEY]},
                        {"status_id": shared_recovery.OrcaStatus.PENDING.value},
                        {"error_message": None},
                        {"request_time": datetime.now(timezone.utc).isoformat()},
                        {"last_update": datetime.now(timezone.utc).isoformat()},
                        {"completion_time": datetime.now(timezone.utc).isoformat()},
                    ]